satellite/audit/queue_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
```

Comment:
the reason for package audit is because it's a unit test, and the the unexported `queue` field on `Queue` can't be instantiated if this package uses `audit_test`.

Reply:
Oh, it doesn't look like the unexported queue field is used except for 
```
q := &Queue{
	queue: []storj.Path{},
}
```
But that can be replaced with `q := &Queue{}`

Reply:
that didn't seem to work because the unexported queue field is still nil

Reply:
oop never mind you were right